resources.py

```diff
@@ -352,15 +352,21 @@ def wait_for_non_tentative_ipv6(self, timeout=60):
         address = str(self.ip('public', 6))
         until = datetime.utcnow() + timedelta(seconds=timeout)
 
+        preferred = re.compile(
+            fr'^\s+inet6 {address}/128 scope global'
```

Comment:
Not sure if it's given that this is always `/128?`. As far as I understand it would also be valid to use the subnet mask here and prefer the GUA instead of the link local address for link local traffic. This might depend on the image used for the test.

I suggest using `[0-9]{2,3}`. 

Also the `scope global` part is new. Not sure how stable this string is across versions. If we want to do fancier output parsing it might be better to use the JSON output format (`ip -json`).